envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterFactory.kt

```diff
@@ -86,16 +85,25 @@ class RBACFilterFactory(
     data class Rules(val shadowRules: RBAC.Builder, val actualRules: RBAC.Builder)
 
     private fun getRules(
-        serviceName: String,
         incomingPermissions: Incoming,
         snapshot: GlobalSnapshot,
         roles: List<Role>
     ): Rules {
 
         val incomingEndpointsPolicies = getIncomingEndpointPolicies(
-                serviceName, incomingPermissions, snapshot, roles
+            incomingPermissions, snapshot, roles
         )
 
+        val incomingWithAnyPrincipalsInLoggedEndpoints = incomingEndpointsPolicies.asSequence()
```

Comment:
I think function `allowLoggedEndpointsPolicy` used in `unlistedAndLoggedEndpointsPolicies ` does the same thing as `unlistedAndLoggedEndpointsPolicies`. Maybe we should change logic in ``unlistedAndLoggedEndpointsPolicies` to always add ``unlistedAndLoggedEndpointsPolicies` instead of this?